src/backend/api/reviews.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const { sum } = require('../database');
+const router = express.Router();
+const knex = require('../database');
+
+// Returns all reviews
+// GET api/reviews/
+router.get('/', async(request, response) => {
+    try {
+        const getAllReviews = await knex('review'); //.select('title');
+        response.json(getAllReviews);
+    } catch (error) {
+        throw error;
+    }
+});
+
+// Adds a new review
+// POST api/reviews/
+router.post('/', async(request, response) => {
+    try {
+        const postReservation = await knex('review').insert({
```

Comment:
Consider doing some validation of this data before you simply drop it into the database!

Reply:
i tried to fix it